src/App.js

```diff
@@ -1,47 +1,66 @@
-import React, { useState, useEffect } from 'react';
-import AddTodoForm from './AddTodoForm';
-import TodoList from './TodoList';
+import React, { useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AddTodoForm from "./AddTodoForm";
+import TodoList from "./TodoList";
+import styles from "./App.module.css";
 
 function App() {
-  const [todoList, setTodoList] = useState([]);
-  const [isLoading, setIsLoading] = useState(true);
+	const [todoList, setTodoList] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-      new Promise((resolve, reject) =>
-        setTimeout(() => {
-          resolve({ data: { todoList: JSON.parse(localStorage.getItem('savedTodoList')) || [] }}
-        )}, 2000))
-      .then((result) => {
-        setTodoList(result.data.todoList)
-        setIsLoading(false)
-      })
-  }, [])
-  
-  const addTodo = (newTodo) => {
-      setTodoList([...todoList, newTodo]);
-  }
+	useEffect(() => {
+		const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`;
+		try {
+			fetch(url, {
+				headers: {
+					Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+				},
```

Comment:
If you ever find that directly applying values to an argument for a function like fetch, you can always create a configuration object and pass it in. For example:
```
const options = {
  method: 'GET',
  headers: {
    Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
  },
};
fetch(url, options)
```
This is a matter of style preference and what you have is completely fine.

Reply:
Thanks!